### PR TITLE
Add registerRef for top level form to access the actual references to the HTML inputs

### LINKED ISSUE #121: Formsy cannot access the actual input refs. This prevents autofocusing to an invalid field with HTML5 form validation turned off
Currently because the `withFormly` HOC doesn't store references to the actual input fields. It's impossible for the top level form to access the actual references to the HTML inputs, unless you reimplement ref registration inside your own component. 

This is a problem because if you want to focus (with `noValidate={true}`) to an invalid field on invalid submit with you cannot access the html nodes to call `.focus()`.

If the HOC withFormly provided a function to store a HTML node (ref) inside itself then the form would have access to everything it needed.

PR coming soon.

### PR DESCRIPTION
**Pull request by TheMcMurder**
This fixes #121

### CODE REVIEW

File: src/Wrapper.js [hunk 1]
```diff
@@ -216,6 +221,7 @@ export default (Component) => {
   };
 
   WrappedComponent.defaultProps = {
+    innerRef: null,
```
Comment (TheMcMurder) [inline src/Wrapper.js]: Fixes the build which is currently broken here.

Same solution as is implemented in #124

**Comment by rkuykendall** [comment]
@TheMcMurder Are you still interested in having this merged into formsy main?

**Comment by TheMcMurder** [comment]
I still believe it's a better API than the existing options specified in #121 so yes.

Without this feature I had to move on from formsy. So I'm not using formsy. I leave it up to the project maintainers to decide if it's worth merging this or not.

**Comment by rkuykendall** [comment]
Thank you, I totally understand and I'm sorry that was the case.

**Comment by rkuykendall** [comment]
I thought it over, and while this is good code with good tests, we shouldn't merge anything unless a piece of code or issue has a "Champion" to advocate for it. I changed the name to be more easily findable in the future in case it comes up again, but I'm going to close it for now. Thanks for your contribution.

**Pull request closed without merging**